bindings/python/mappings.py

```diff
@@ -228,14 +229,25 @@ def lowest_costs(self, nresults=5):
         result = []
         n = 0
         for inputs in self.input_routes:
-            owncost = 1
-            for cost, idx in get_lowest_costs(inputs, nresults=nresults):
-                if isinstance(self.cost, Callable):
-                    values = get_values(inputs, idx, magnitudes=True)
-                    owncost = self.cost(**values)
+            costs = np.array([0.0])
+            for input in inputs.values():
+                if isinstance(input, MappingStep):
```

Comment:
Can you add a comment on how costs are calculated for MappingStep?

Reply:
Done